src/Entities/RulesEngine/Actions/Blacklist.php

```diff
@@ -0,0 +1,95 @@
+<?php
+/**
+ * This source file is proprietary and part of Rebilly.
+ *
+ * (c) Rebilly SRL
+ *     Rebilly Ltd.
+ *     Rebilly Inc.
+ *
+ * @see https://www.rebilly.com
+ */
+
+namespace Rebilly\Entities\RulesEngine\Actions;
+
+use DomainException;
+use Rebilly\Entities\RulesEngine\RuleAction;
+
+/**
+ * Class Blacklist.
+ */
+final class Blacklist extends RuleAction
+{
+    public const UNEXPECTED_TYPE = 'Unexpected type. Only %s types are supported';
+
+    public const TYPE_CUSTOMER_ID = 'customer-id';
+
+    public const TYPE_EMAIL = 'email';
+
+    public const TYPE_FINGERPRINT = 'fingerprint';
+
+    public const TYPE_IP_ADDRESS = 'ip-address';
+
+    public const TYPE_PAYMENT_CARD_ID = 'payment-card-id';
```

Comment:
```suggestion
    public const TYPE_PAYMENT_CARD = 'payment-card';
```

Reply:
There is not such blacklist type `payment-card-id`

Reply:
Our spec has `payment-card-id` in it https://rebilly.github.io/RebillyUserAPI/#tag/Rules/paths/~1events~1{eventType}~1rules/get

Reply:
Then its wrong, we never had such type.

Reply:
https://github.com/Rebilly/backend/blob/master/src/Rules/Domain/Actions/Blacklist.php#L22
https://github.com/Rebilly/backend/blob/master/src/Modules/Blacklists/BlacklistType.php#L31